app/main.py

```diff
@@ -1,3 +1,44 @@
+from __future__ import annotations
+
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: int) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, addentum: object | int) -> Distance:
```

Comment:
object is still in type annotations, user more descriptive type annotations, please
object type means that method could accept any object in python (should accept only Distance/int objects)
fix this everywehre